torchdynamo/symbolic_convert.py

```diff
@@ -645,6 +645,24 @@ def LOAD_ATTR(self, inst):
                     self.push(UserMethodVariable(method, obj, **options))
                 else:
                     unimplemented("nn.Module callable")
+            elif isinstance(subobj, list):
+                # If the list contains exclusively nn.Modules, transform
+                # each module into a torchdynamo.NNModuleVariable
+                l = []
+                for i, item in enumerate(subobj):
+                    if isinstance(item, torch.nn.Module):
+                        key = f"{name}_{i}"
```

Comment:
```suggestion
                        key = f"{obj.module_key}.{name}_{i}"
```
To handle the somewhat unlikely case of:
`a.foo` and `b.foo`

Reply:
Good point, thanks!

Reply:
Big oops, I remembered why I didn't have this in the first place: the `get_attr` call with this key breaks the `LayerList` test. (Sending you a paste on Workplace.)